### ISSUE TITLE
Theme changes have no effect on iOS 13 with dark mode enabled

### ISSUE THREAD

**Issue by alex-spataru** [jpnurmi/statusbar#8]
Hi!

While testing my application (which makes use of your project), I found out that if dark mode is enabled on the iOS device, the theme setting is completely ignored and defaults to dark mode (white status bar icons).

In order to fix this, I made the following modifications to [statusbar_ios.mm](https://github.com/jpnurmi/statusbar/blob/master/src/statusbar_ios.mm#L47): 

```
// Includes and @interface

#define SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(v)  ([[[UIDevice currentDevice] systemVersion] compare:v options:NSNumericSearch] != NSOrderedAscending)

// Other functions, etc

static UIStatusBarStyle statusBarStyle(StatusBar::Theme theme)
{
    if (theme == StatusBar::Dark)
        return UIStatusBarStyleLightContent;

    else if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"13.0"))
        return UIStatusBarStyleDarkContent;

    else
        return UIStatusBarStyleDefault;
}
```

However, while my solution works for my test-scenario, I really doubt that my fix is the most elegant or appropriate (I never really worked with iOS/Xcode/Objective-C before). By the way, the `SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO` definition was copy-pasted from [StackOverflow](https://stackoverflow.com/questions/3339722/how-to-check-ios-version).

Greetings!

**Comment by jpnurmi** [comment]
Thanks for the report. Feel free to create a PR with the proposed solution. Unfortunately, I don't have access to a development platform anymore that would allow me to experiment with this. :(

**Comment by jpnurmi** [comment]
Fixed by #9
